shuup_tests/conftest.py

```diff
@@ -54,6 +54,9 @@ def enable_db_access(db):
     from django.utils.translation import activate
     activate("en")
 
+    from django.core.cache import cache
```

Comment:
It think you should clear the shuup cache instead. `from shuup.core import cache`.

Reply:
I clear the Shuup cache above by re-creating it. I have to clear this cache because of 3rd party apps that might be using cache, like Parler. You can see fail in this test log: https://travis-ci.org/shuup/shuup/jobs/403915722